[docs] [voyager] added arguments and commands to yb-voyager-cli for 0.81 (#14605)
yugabyte/yugabyte-db: docs/content/preview/migrate/yb-voyager-cli.md

        --source-db-user username \
        --source-db-password password \
        --source-db-name dbname \
        --source-db-schema schemaName \ #Not applicable for MySQL.
        --oracle-db-sid string \ #Applicable only for Oracle
        --oracle-home string \ #Applicable only for Oracle
        --table-list string \
        --exclude-table-list string
```

### export data status

Get the status report of an ongoing or completed data export operation.

#### Syntax

```sh
yb-voyager export data status [ <arguments> ... ]
```

- *arguments*: See [Arguments](#arguments)

#### Example

```sh
yb-voyager export data status --export-dir /path/to/yb/export/dir
```

> ### import schema

Reviewer: The new description of import schema should read:

During a typical migration workflow, the user is required to run the `import schema` command twice: first **without** the `post-import-data` flag and second time **with** the flag.
The second invocation **must** be done **after** the **data** import is already complete. The second invocation creates indexes and triggers in the target schema.

Default: '\t' (tab); can be changed to comma(,), pipe(|) or any other character.

### --has-header

This argument is to be specified only for CSV file type.

Default: false; change to true if the CSV file contains column names as a header.

**Note**: Boolean flags takes arguments in the format `--flag-name=[true|false]`, and not `--flag-name [true|false]`.

### --file-opts

Comma-separated string options for CSV file format. The options can include the following:

> 1. `escape_char`: escape character

Reviewer 2: do they need to be in a specific order?

Reviewer 3: Any random order would work. Internally identifies and use it accordingly

Reviewer 2: Use bullets then

1. `quote_char`: character used to quote the values

> Default: double quotes '"'

Reviewer 2: is this working the way you intend?
is there a default escape character?

Reviewer 3: thanks for pointing it out @ddhodge
1. Yes, there is a default escape character, it is also double quotes.
2. Also, We should mention somewhere that escape_char and quote_char are only valid/required for CSV file format(which is the default format)

Author: @sanyamsinghal , what are the other file formats that we need to mention apart from CSV. Are others supported?

Reviewer 3: Supported formated are: csv and text

Default: '\t' (tab); can be changed to comma(,), pipe(|) or any other character.

### --has-header

This argument is to be specified only for CSV file type.

Default: false; change to true if the CSV file contains column names as a header.

**Note**: Boolean flags takes arguments in the format `--flag-name=[true|false]`, and not `--flag-name [true|false]`.

### --file-opts

Comma-separated string options for CSV file format. The options can include the following:

- `escape_char`: escape character

- `quote_char`: character used to quote the values

> Default (for both options): double quotes '"'

Reviewer 2: ```suggestion
Default: double quotes (") for both escape and quote characters
```

> Note that `escape_char` and `quote_char` are only valid/required for CSV file format.

Reviewer 2: ```suggestion
Note that `escape_char` and `quote_char` are only valid and required for CSV file format.
```

> Default : 100,000

Reviewer 2: ```suggestion
Default: 100,000
```

Default: '\t' (tab); can be changed to comma(,), pipe(|) or any other character.

### --has-header

This argument is to be specified only for CSV file type.

Default: false; change to true if the CSV file contains column names as a header.

> **Note**: Boolean flags takes arguments in the format `--flag-name=[true|false]`, and not `--flag-name [true|false]`.

Reviewer 2: ```suggestion
**Note**: Boolean flags take arguments in the format `--flag-name=[true|false]`, and not `--flag-name [true|false]`.
```

### --file-opts

Comma-separated string options for CSV file format. The options can include the following:

- `escape_char`: escape character

- `quote_char`: character used to quote the values

> Default : double quotes (") for both escape and quote characters

Reviewer 2: ```suggestion
Default: double quotes (") for both escape and quote characters
```

Note that `escape_char` and `quote_char` are only valid and required for CSV file format.

Example: `--file-opts "escape_char=\",quote_char=\""` or `--file-opts 'escape_char=",quote_char="'`

### --format

Specifies the format of your data file with CSV or text as the supported formats.

> Default : CSV

Reviewer 2: ```suggestion
Default: CSV
```

### --post-import-data

Run this argument with [import schema](#import-schema) command to import indexes and triggers after data import is complete.

> The `--post-import-data` argument assumes that data import is already done and imports only indexes and triggers.

Reviewer 4: ```
When `--post-import-data` flag is passed, the `import schema` command imports indexes and triggers in the target DB.
```

### --oracle-db-sid

Oracle System Identifier (SID) you can use while exporting data from Oracle instances.

### --oracle-home

Path to set `$ORACLE_HOME` environment variable. `tnsnames.ora` is found in `$ORACLE_HOME/network/admin`.

### --use-orafce

Enable using orafce extension in export schema.

> Default : true

Reviewer 2: ```suggestion
Default: true
```
